packages/synapse-react-client/.storybook/preview.jsx

```diff
@@ -113,8 +114,16 @@ initialize({
   },
 })
 
+const fontLoader = async () => ({
+  fonts: await Promise.all([document.fonts.load('700 1em Lato')]),
+})
+
 export const loaders = [mswLoader]
 
+if (isChromatic && document.fonts) {
+  loaders.push(fontLoader)
+}
```

Comment:
When deployed to Chromatic, wait for the font to load before rendering the story. This should make our screenshots more stable